src/components/DocsHub/styles.css

```diff
@@ -179,3 +179,100 @@
   margin-bottom: 3px;
   display: flex;
 }
+
+
+
```

Comment:
Why have we added this CSS?

Reply:
That fixed the css issue I was having earlier in the docs section. It's like a contigency for the styling cause docusaurus sometimes changes the classnames in the builds and these are some of the ones for the gh-pages build